spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestViews.java

```diff
@@ -62,6 +64,9 @@ public class TestViews extends ExtensionsTestBase {
   private static final Namespace NAMESPACE = Namespace.of("default");
   private final String tableName = "table";
 
+  @Parameter(index = 3)
+  protected String location;
```

Comment:
Different catalog has different scheme+path. Current tests were only for InMemoryCatalog.